### PR TITLE
python bindings for create_metric_report

### PR DESCRIPTION
**Pull request by Krovatkin**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #79679
* #79678

### CODE REVIEW

File: torch/_lazy/metrics.py [hunk 1]
```diff
@@ -11,3 +11,7 @@ def counter_names():
 def counter_value(name: str):
     """Return the value of the counter with the speficied name"""
     return torch._C._lazy._counter_value(name)
+
+def create_metric_report():
```
[the remark this answers is not in the document]
Comment (Krovatkin) [reply torch/_lazy/metrics.py:15]: sure I'll make a change

**Review by JackCaoG — APPROVED** [review]
Thanks!

**Review by wconstab — APPROVED** [review]
i'd prefer if you change all 3 to `metrics_report` so its consistent and matches XLA api, i think you changed to `metric_report` instead of `metrics_report` by mistake?  but i'll stamp to unblock

**Pull request closed without merging**